Formula/gnuradio.rb

```diff
@@ -12,7 +12,15 @@ class Gnuradio < Formula
     sha256 "41eb9fdae72761b7a83f284f1e7613da9e3ce916e0f98f20ad3aafe417be5a4e" => :yosemite
   end
 
+  # Fixes linkage of python (swig) bindings directly to python
+  # https://github.com/gnuradio/gnuradio/pull/1146
+  patch do
+    url "https://github.com/gnuradio/gnuradio/pull/1146.patch"
+    sha256 "385b62cc46ff07696f7ff4ff89bbb4891909947027f9712be040937d477f6d81"
+  end
+
   option :universal
+  option "with-documentation", "Build with documentation"
```

Comment:
Instead call this `with-doxygen` and make it `[:build, :optional]` and, if necessary, add a `deprecated_option`.